### DISCUSSION TITLE
Changing the license

### DISCUSSION THREAD

**Discussion by tr7zw** [tr7zw/ItemSwapper#116] [General]
Hey there,
I recently got notified about https://blog.modrinth.com/p/licensing-guide . Right now this project uses the "GNU General Public License v3.0" license(short [GPL-3.0](https://spdx.org/licenses/GPL-3.0-only.html)), but as outlined by the article

> Typically, when a copyleft license is wanted, the [GPL-3.0](https://spdx.org/licenses/GPL-3.0-only.html) or [AGPL-3.0](https://spdx.org/licenses/AGPL-3.0-only.html) would be used. However, these licenses are incompatible if linking into Minecraft, due to an issue with the [difference between proprietary and free software](https://www.gnu.org/licenses/gpl-faq.html#GPLPlugins) outlined by these licenses.

, using GPL-3.0 may technically not be usable for a Minecraft Mod, since Minecraft is closed source.
In general, this isn't a giant issue, but it would be better to resolve this long-term (and especially sooner than later).
The plan is to switch to the [LGPL-3.0](https://spdx.org/licenses/LGPL-3.0-only.html) license, which is [mostly](https://developer.mozilla.org/en-US/docs/Glossary/LGPL) the same as the current one but allows linking to proprietary software(Minecraft).
Please give a quick confirmation under this discussion: 
@AdvancedWipe ✅
@RedVortexDev 
@Hexasan ✅
@TheMrEngMan ✅
@Mccandycube6623 
@ViTulyankin ✅
@ttrafford7 
@fartdev ✅
@spnda ✅
@BeansBeefBroccoli ✅
@GodGun968 ✅

**Comment by spnda** [comment]
I guess this also stands for all other fabric/forge mods of yours (Spigot plugins on the other hand have to be GPL-3.0), but yeah I'm fine with it (and for all other mods I contributed to).

**Reply by tr7zw** [reply]
AFAIK this is my only project using GPL-3 (all others are MIT/my own license/ARR), so only this project is affected I think.

**Comment by TheMrEngMan** [comment]
All good with me, thanks for the mention.

**Comment by AdvancedWipe** [comment]
I am fine with that as well.

**Comment by BeansBeefBroccoli** [comment]
Sounds good 👍

**Comment by Hexasan** [comment]
Ok with me👍

**Comment by Vi-Tul** [comment]
No problem 👌

**Comment by fartdev** [comment]
No problem

**Comment by GodGun968** [comment]
OK

**Comment by tr7zw** [comment]
Mainly @RedVortexDev is missing(they changed their name, and probably didn't get the ping correctly). The other missing ones are language files, which can be removed if the authors are against the change(for whatever reason).
